cmd/config/testdata/help.golden

```diff
@@ -10,7 +16,7 @@ Flags:
       --unset string   Unset a config field
```

Comment:
Minor (and I know, I'm tagging this on a test file) but I noticed that `--unset` is marked as taking a string parameter whereas `--set` isn't marked to have any parameter. It should also have `string`.

Reply:
https://app.shortcut.com/launchdarkly/story/242359/config-set-should-say-it-takes-a-string